dms/models/dms_file.py

```diff
@@ -411,10 +411,7 @@ def _compute_extension(self):
     @api.depends("name", "content")
     def _compute_mimetype(self):
         for record in self:
-            binary = base64.b64decode(record.with_context({}).content or "")
-            record.res_mimetype = guess_mimetype(
-                binary, default="application/octet-stream"
-            )
+            record.res_mimetype = guess_mimetype(record.content or b"")
```

Comment:
Good for me, just a question, why the name on the depends?

Reply:
In my opinion I think that `name` field is not necessary because it is not really used to generate the `res_mimetype` field BUT as it was already like this (`name` field as depends) in the code I have not removed it.
What do you think @pedrobaeza  about it?

Reply:
Maybe it was a trick for getting that recomputed, but it shouldn't be needed.